### DISCUSSION TITLE
DingTalk QR code is now available

### DISCUSSION THREAD

**Discussion by Ziy1-Tan** [alibaba/ilogtail#642] [Help] [answered]
Hi! I was new and I want to make contribution in the future. but QR code is now available.
https://ilogtail.gitbook.io/ilogtail-docs/about/readme#lian-xi-wo-men
Could any one invite me in or provide a new code, thx!

**Comment by messixukejia** [comment, marked as answer]
It is updated on github home page. https://github.com/alibaba/ilogtail

![image](https://user-images.githubusercontent.com/13250519/218013675-f2ebb773-d20e-4356-bd8c-c7103948e486.png)

Gitbook will be updated for next time the code is synchronized
